src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3283,6 +3283,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </ItemGroup>
 
     <PropertyGroup>
+      <GenerateResourceWarnOnBinaryFormatterUse Condition="'$(GenerateResourceWarnOnBinaryFormatterUse)' == ''">false</GenerateResourceWarnOnBinaryFormatterUse>
```

Comment:
I don't think I'd explicitly set a default here; I'd just pass it undefined in general.